app/assets/stylesheets/views/_specialist-document.scss

```diff
@@ -1,3 +1,33 @@
+$contents-link-background: #dee0e2;
```

Comment:
This colour is already defined in the colour styleguide as $panel-colour (http://govuk-elements.herokuapp.com/colour/)